package-system/sqlite/FindSQLite.cmake

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# this file actually ingests the library and defines targets.
+set(TARGET_WITH_NAMESPACE "3rdParty::SQLite")
+if (TARGET ${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+
+set(MY_NAME "SQLite")
+
+set(${MY_NAME}_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/SQLite)
+set(${MY_NAME}_LIBS_DIR ${CMAKE_CURRENT_LIST_DIR}/SQLite/lib)
+
+set(${MY_NAME}_LIBRARY_DEBUG   ${${MY_NAME}_LIBS_DIR}/debug/${CMAKE_STATIC_LIBRARY_PREFIX}sqlite3${CMAKE_STATIC_LIBRARY_SUFFIX})
+set(${MY_NAME}_LIBRARY_RELEASE ${${MY_NAME}_LIBS_DIR}/release/${CMAKE_STATIC_LIBRARY_PREFIX}sqlite3${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+# we set it to a generator expression for multi-config situations:
+set(${MY_NAME}_LIBRARY  
+    "$<$<CONFIG:profile>:${${MY_NAME}_LIBRARY_RELEASE}>"
+    "$<$<CONFIG:Release>:${${MY_NAME}_LIBRARY_RELEASE}>"
+    "$<$<CONFIG:Debug>:${${MY_NAME}_LIBRARY_DEBUG}>")
```

Comment:
The `${MY_NAME}_LIBRARY` variable in the CMake Find scripts are no longer needed.
Because we specify the target type correctly to be STATIC, we can use the `IMPORTED_LOCATION` property to set the library location.